pkg/cmd/codespace/jupyter.go

```diff
@@ -44,13 +44,13 @@ func (a *App) Jupyter(ctx context.Context, codespaceName string) (err error) {
 	}
 	defer safeClose(session, &err)
 
+	a.StartProgressIndicatorWithLabel("Starting JupyterLab on codespace")
```

Comment:
Moved this here because if we can't connect to the grpc port for a while, there's no progress indicator spinning and it seemed more appropriate to be a part of the actual command progress rather than the starting codespace progress